### DISCUSSION TITLE
If a certificate has the ip address, how is dhcp handled

### DISCUSSION THREAD

**Discussion by jmls** [slackhq/nebula#1121] [General]
Nebula Noob here - from what I've read, the client certificate needs to have the ip address within the certificate. I was wondering how a change in IP address, for example when a lease expires, is handled. Does that machine go offline until someone creates a new certificate ? Or is there an automated process to handle this ?

I may be overlooking something completely obvious and will look stupid, but had to ask ;) 

Curious minds want to know.

**Comment by johnmaguire** [comment]
Hi @jmls - the IP contained in the certificate is the Nebula VPN IP. This is a static IP assigned to the host. There is no need to put the local network IP address in the cert.

Hope that helps!

**Comment by jmls** [comment]
that makes much more sense :) thanks !
